styles/style.css

```diff
@@ -0,0 +1,687 @@
+  font-weight: 600;
+  margin: 0 auto;
+  padding-bottom: 16px;
+  position: relative;
+  top: 11px;
+  left: 9px;
+  letter-spacing: .6px;
+}
+
+.title-subtext {
+  font-size: 15px;
+  font-weight: 500;
+  font-style: normal;
+  margin: 0 auto;
+  position: relative;
+  bottom: 12px;
+  left: 9px;
+}
+
+.car-tesla {
+  width: 68%;
+  margin: 0 auto;
+  margin-bottom: 38px;
+  position: relative;
+  top: 15px;
+  right: 24px;
+}
+
+.stat {
+  width: 700px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  margin-bottom: 53px;
+}
+
+.params {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  padding: 0 300px 0 382px;
+}
+
+.wrap-params-one {
+  display: flex;
+  align-items: center;
+}
+
+.stat__speed {
+  width: 46px;
+  height: 35px;
+  margin-right: 187px;
+}
+
+.stat-power {
+  width: 46px;
+  height: 35px;
+  margin-right: 214px;
+}
+
+.wrap-params-two {
+  display: flex;
+  align-items: center;
+}
+
+.stat__test-drive {
+  width: 102px;
+  height: 0;
+  margin-right: 115px;
+  position: relative;
+  right: 12px;
+}
+
+.test-drive__red {
+  display: none;
+}
+
+.stat__btn {
+  position: relative;
+  top: 11px;
+  right: 69px;
+}
+
+.btn__first-floor {
+  text-transform: uppercase;
+  display: block;
+  border: 2px solid #e51937;
+  color: #e51937;
+  font-weight: bold;
+  font-size: 12px;
+  font-style: normal;
+  letter-spacing: .5px;
+  background: #fff;
+  padding: 10px 41px;
+  transition-duration: .4s;
+  cursor: pointer;
+}
+
+.btn__first-floor:hover {
+  color: #fff;
+  background: #e51937;
+  border-color: #e51937;
+}
+
+.swipe-down_btn {
+  position: relative;
+  left: 8px;
+  transition-duration: .9s;
+}
+
+.swipe-down_btn:hover {
+  top: 5px;
+}
+
+.advantage-info {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background-image: url(../img/2013-tesla-model-s-cockpit.png);
+  background-position: bottom left;
+  background-repeat: no-repeat;
+}
+
+.top-title {
+  color: #e51937;
+  font-size: 70px;
+  font-weight: 600;
+  line-height: 85px;
+  margin: 40px 0 0 40px;
+}
+
+.info__description-advantage {
+  max-width: 423px;
+  color: #fff;
+  background: #e51937;
+  padding: 55px 45px;
+}
+
+.advantage-text {
+  font-size: 30px;
+  line-height: 37px;
+  font-weight: 600;
+  margin: 0;
+}
+
+.advantage-subtext {
+  font-size: 15px;
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 55px;
+}
+
+.image-advantage {
+  margin: 0 0 160px 0;
+}
+
+.advantage_info-btn {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  white-space: nowrap;
+}
+
+.info_btn-text {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 13px;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.info-btn__img {
+  margin-right: 5px;
+}
+
+.btn-info {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  padding: 10px 43px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)